challenges/prototypes.js

```diff
@@ -6,28 +6,44 @@
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
 
+function CuboidMaker(cubAttr) {
+  this.length = cubAttr.length;
+  this.width = cubAttr.width;
+  this.height = cubAttr.height;
+}
 
 /* == Step 2: Volume Method ==
   Create a method using CuboidMaker's prototype that returns the volume of a given cuboid's length, width, and height
   
   Formula for cuboid volume: length * width * height
 */
 
+CuboidMaker.prototype.volume = function() {
+  console.log(`${this.length} * ${this.width} * ${this.height}`);
+};
 
```

Comment:
You can simply wrap string literals within the same braces, for example: `${this.length * this.width * this.height}`. This will save some time in typing and make the code look cleaner